lessons/pydata/api/index.ipynb

```diff
@@ -0,0 +1,2168 @@
+{
```

Comment:
<p>Chceme to tam mít, i když v Pythonu nic moc podobného <code>undefined</code>  nemáme? Klidně by se to dalo předělat na ukázku rozdílu mezi <code>0</code>  a <code>None</code> , ale v této podobě by to možná více mátlo než vysvětlovalo.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>undifined pryč</p>